engines/cge/talk.cpp

```diff
@@ -88,6 +91,15 @@ Talk::Talk(CGEEngine *vm)
 	_wideSpace = false;
 }
 
+const char *lastText = "";
```

Comment:
It might be better here to initialise `lastText` to nullptr. You could also possibly make it a member of the `Talk` class.